File: app-insert-menu/python/getMenuBiru.py
import pymssql
import sys

mydb = pymssql.connect(
    host="10.10.101.96",
    port="1433",
    user="appreader",
    password="Simpang4244",
    database="MPMIT"
)

mycursor=mydb.cursor()

sql = "SELECT * FROM MPMIT.dbo.MPM_MENU"
mycursor.execute(sql)

result = ""

for table_name in mycursor:
    result += "<tr>"
    result += ("<td>" + table_name[0] + "</td>")
    result += ("<td>" + table_name[1] + "</td>")
    result += ("<td>" + table_name[2] + "</td>")
    result += ("<td>" + str(table_name[3]) + "</td>")
    result += ("<td>" + table_name[4] + "</td>")
    result += ("<td>" + table_name[5] + "</td>")
    result += ("<td>" + table_name[6] + "</td>")
    result += ("<td>" + table_name[7] + "</td>")
    result += "</tr>"

print(result)

File: app-insert-menu/php/getMenuBiru.php
<?php 
    // include '../../assets/php/connection.php';

    // $dbname = "MPMIT";

    // $con = mysqli_connect($servername, $username, $password, $dbname);

    // if (!$con) {
    //     die("Connection failed: " . mysqli_connect_error());
    // }
    // echo "Connected successfully";

    // $sql = "SELECT * FROM MPMIT.dbo.MPM_MENU";
    // $result = mysqli_query($con, $sql);

    $res = "<thead>";
    $res .= "<tr>";
    $res .= '<th class="th-sm"> MENU ID </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> CATEGORY ID </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> CATEGORY GROUP ID </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> ORDERED NUMBER </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> PROCESS GROUP NAME </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> MODULE NAME </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> MENU NAME </th>';
    $res .= "</tr>";

    $res .= "<tr>";
    $res .= '<th class="th-sm"> PATH </th>';
    $res .= "</tr>";

    $res .= "</thead>";

    $res .= "<tbody>";

    if (mysqli_num_rows($result) > 0) {
        while($row = mysqli_fetch_assoc($result)) {
            $res .= "<tr>";
            $res .= "<td>".$row['MENU_ID']."</td>";
            $res .= "<td>".$row['CATEGORY_ID']."</td>";
            $res .= "<td>".$row['CATEGORY_GROUP_ID']."</td>";
            $res .= "<td>".$row['ORDERED_NUMBER']."</td>";
            $res .= "<td>".$row['PROCESS_GROUP_NAME']."</td>";
            $res .= "<td>".$row['MODULE_NAME']."</td>";
            $res .= "<td>".$row['MENU_NAME']."</td>";
            $res .= "<td>".$row['PATH']."</td>";
            $res .= "</tr>";
        }
    }

    $res .= "</tbody>";

    // $conn->close();

    echo $res;
?>